otter/integration/tests/test_lbsh.py

```diff
@@ -97,3 +98,80 @@ def test_oob_deleted_clb_node(self):
             ),
             timeout=timeout_default
         )
+
+    @tag("LBSH-002")
+    @inlineCallbacks
+    def test_move_node_to_oob_lb(self):
+        """
+        1 group, LB1 in config, LB2 not in any autoscale configs:
+
+        Server node moved from LB1 to LB2
+        Assert: Server put back on LB1
+        Assert: Server removed from LB2
+        """
+        # Create another loadbalancer not to be used in autoscale
+        # The CLB will not be added to the helper, since when the helper
+        # creates a group, it automatically adds the clb
+        clb_other = CloudLoadBalancer(pool=self.helper.pool)
+
+        yield clb_other.start(self.rcs, self)
+        yield clb_other.wait_for_state(
+            self.rcs, "ACTIVE", timeout_default)
+
+        clb_as = self.helper.clbs[0]
+
+        nodes_as = yield clb_as.list_nodes(self.rcs)
+        nodes_other = yield clb_other.list_nodes(self.rcs)
+        self.assertEqual(len(nodes_as['nodes']), 0,
+                         "There should be no nodes on the CLB yet.")
+        self.assertEqual(len(nodes_other['nodes']), 0,
+                         "There should be no nodes on the CLB yet.")
+
+        group, _ = self.helper.create_group(min_entities=1)
+        yield self.helper.start_group_and_wait(group, self.rcs)
+
+        nodes_as = yield clb_as.list_nodes(self.rcs)
+        nodes_other = yield clb_other.list_nodes(self.rcs)
+        self.assertEqual(
+            len(nodes_as['nodes']), 1,
+            "There should be 1 node on the autoscale CLB.")
+        self.assertEqual(
+            len(nodes_other['nodes']), 0,
+            "There should still be 0 nodes on the out of band CLB")
+
+        the_node = nodes_as["nodes"][0]
+        node_info = {
+            "address": the_node["address"],
+            "port": the_node["port"],
+            "condition": the_node["condition"],
+            "weight": 2
+        }
+
+        yield clb_as.delete_nodes(self.rcs, [the_node['id']])
+        yield clb_other.add_nodes(self.rcs, [node_info])
+
+        nodes_as = yield clb_as.list_nodes(self.rcs)
+        nodes_other = yield clb_other.list_nodes(self.rcs)
+        self.assertEqual(clb_as.wait_for_nodes(
```

Comment:
This should probably just be `clb_as.wait_for_nodes(self.rcs, HasLength(0), timeout=timeout_default)`, since that would error anyway if there aren't the requisite number of nodes.